### PR TITLE
Remove metadata creation from central state

### PR DESCRIPTION
**Pull request by martincik**
We already have a SPL token in production with
initialized metadata so we have to remove the metadata init code from our central state.

### CODE REVIEW

File: smart-contract/program/tests/functional.rs [hunk 1]
```diff
@@ -62,43 +65,78 @@ async fn test_staking() {
...
+    ////
+    // Metadata creation
+    ////
+    let create_metadata_ix = create_metadata_accounts_v3(
+        mpl_token_metadata::ID,
+        metadata_key,
+        mint,
+        authority.pubkey(),
+        prg_test_ctx.payer.pubkey(),
+        authority.pubkey(),
```
Comment (dr497) [inline smart-contract/program/tests/functional.rs:87]: `central_state`

File: smart-contract/program/tests/functional.rs [hunk 2]
```diff
@@ -62,43 +65,78 @@ async fn test_staking() {
...
+        "Access Protocol".to_string(),
+        "ACS".to_string(),
+        "URI".to_string(),
+        None,
+        0,
+        true,
```
Comment (dr497) [inline smart-contract/program/tests/functional.rs:93]: Should be `false`

**Comment by martincik** [comment]
@dr497 Thank you for the suggestions, yet I couldn't get it to work. Would you mind trying it yourself and helping me out? I tried a lot of things, but couldn't make it work. :( Thank you.

**Comment by dr497** [comment]
> @dr497 Thank you for the suggestions, yet I couldn't get it to work. Would you mind trying it yourself and helping me out? I tried a lot of things, but couldn't make it work. :( Thank you.

I confirm that `b1473645d9761cb7bc0604fb9e985add35108f6e` is working on my end. What version of metaplex metadata do you load in program test?

**Pull request merged**